Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -21,7 +21,66 @@ private let commentLinePrefixCharacterSet: NSCharacterSet = {
   return characterSet
 }()
 
+private var keyByteOffsetCache = "ByteOffsetCache"
+
 extension NSString {
+    /**
+    ByteOffsetCache is cache byte offset and UTF8Index of UTF16 based location.
+    This depends on using for that will be calculated from begening of the string to the end.
+    */
+    @objc private class ByteOffsetCache: NSObject {
+        struct Entry {
```

Comment:
`Entry` isn't a very descriptive name in this case. `ByteOffsetIndexPair` could work.